File: SAMPLE/MYSQL/CS/blog_type.cs
public class SECTION
{
    public ulong
        Id,
        Number;
    public string
        Slug,
        Name,
        Text,
        Image;
    public ulong
        ImageIndex;
}

// ~~

public class USER
{
    public ulong
        Id;
    public string
        FirstName,
        LastName,
        Email,
        Pseudonym,
        Password,
        Phone,
        Street,
        City,
        Code,
        Region,
        Country,
        Company;
    public bool
        ItIsAdministrator;
}

// ~~

public class ARTICLE
{
    public ulong
        Id,
        SectionId,
        UserId;
    public string
        Slug,
        Title,
        Text,
        Image;
    public DateTime
        Date;
    public SECTION
        Section;
    public USER
        User;
    public ulong
        ImageIndex;
}

// ~~

public class COMMENT
{
    public ulong
        Id,
        ArticleId,
        UserId;
    public string
        Text;
    public DateTime
        DateTime;
    public ARTICLE
        Article;
    public USER
        User;
}

// ~~

public class SUBSCRIBER
{
    public ulong
        Id;
    public string
        Name,
        Email;
}

File: SAMPLE/POSTGRESQL/RS/blog_type.rs
struct SECTION
{
    id : u64,
    number : u64,
    slug : String,
    name : String,
    text : String,
    image : String,
    image_index : u64
}

// ~~

struct USER
{
    id : u64,
    first_name : String,
    last_name : String,
    email : String,
    pseudonym : String,
    password : String,
    phone : String,
    street : String,
    city : String,
    code : String,
    region : String,
    country : String,
    company : String,
    it_is_administrator : bool
}

// ~~

struct ARTICLE
{
    id : u64,
    section_id : u64,
    user_id : u64,
    slug : String,
    title : String,
    text : String,
    image : String,
    date : String,
    section : Weak<SECTION>,
    user : Weak<USER>,
    image_index : u64
}

// ~~

struct COMMENT
{
    id : u64,
    article_id : u64,
    user_id : u64,
    text : String,
    date_time : String,
    article : Weak<ARTICLE>,
    user : Weak<USER>
}

// ~~

struct SUBSCRIBER
{
    id : u64,
    name : String,
    email : String
}
